### ISSUE TITLE
Merge command cogs

### ISSUE THREAD

**Issue by tangalbert919** [sayori34/dokis#74]
- [ ] Depends on #73 

We need to merge the command cogs that are not specific to any character (which is most of the commands, actually). However, doing this depends on another issue being completed first.

**Comment by tangalbert919** [comment]
Yuri's command cogs will have to be merged last, due to the fact that the whole "Act 1/2" thing is being moved to the database.
